Updated creating your first artboard Indigo.Design System section (#371)
IgniteUI/indigo-design-docfx: en/creating-an-artboard.md

## Creating Your First Artboard

> At the end of this topic you'll have a Sketch Artboard for a registration screen using Indigo Design System.

Reviewer: ```suggestion
At the end of this topic you'll have a Sketch Artboard for a registration screen using the **Indigo.Design System**.
```

## What you need

> If the [Titillium Web Font][1] isn't installed on your computer, please do it before starting. All texts in the Indigo Design System use it.

Reviewer: ```suggestion
If the [Titillium Web Font][1] isn't installed on your computer, please do it before starting. All texts in the **Indigo.Design System** use it.
```

## Step by step

> ### 1. Add the Indigo Design System in Sketch.

Reviewer: ```suggestion
### 1. Add the **Indigo.Design System** in Sketch.
```

> 1.  Download the [Indigo Design System](https://cloud.indigo.design/home).

Reviewer: ```suggestion
1.  Download the [**Indigo.Design System**](https://cloud.indigo.design/home).
```

>     It contains 1 Sketch file with 26 Pages in the Pages section which are distinguished by three types of icons:

Reviewer: ```suggestion
    It contains a Sketch file with 26 Pages in the Pages section which are distinguished by three types of icons:
```

>     - **1-5 Pages**  

Reviewer: ```suggestion
    - **🎨 ~** styling pages 
```

>       All the styling assets such as Material Icons, Colors, Typography, Elevation, and Illustrations.

Reviewer: ```suggestion
      All the styling assets such as Material Icons, Colors, Typography, Elevation, and Illustrations can be found in the five pages on top.
```

>     - **6 Page**  

Reviewer: ```suggestion
    - **🧩 Components**  page 
```

      A vast set of Components to layout your design ideas.  

>       Through the overrides section you can choose from numerous presets compatible with Ignite UI for Angular and our code generation engine.

Reviewer: ```suggestion
      Through various presets available in the Sketch insert menu and overrides for states and layouts to configure in the right panel of the Sketch UI, you can create layouts compatible with Ignite UI for Angular and our code generation engine.
```

>     - **7-26 Pages**  

Reviewer: ```suggestion
    - **🌆 ~** patterns pages
```

>        The components shape up meaningful patterns for productive application design.

Reviewer: ```suggestion
       The components shape up meaningful patterns for productive application design that can be found spread across the remaining pages.
```

2.  Open up the Sketch app on your Mac, click on the Sketch item from the Apple Menu, and select Preferences.

3.  In the dialog, select the **Libraries** tab and click the **Add Library...** button.

4.  Navigate to the folder where you have downloaded the library.

5.  Select it and click **Open**.

6.  Make sure it shows up in the list in the **Preferences** dialog and that it is checked.

  <div class="divider--half"></div>

>   <img class="responsive-img" src="images/Creating_An_Artboard_2.png" srcset="images/Creating_An_Artboard_2@2x.png" />

Reviewer: Image looks too big maybe we can downsize it in half

  <div class="divider--half"></div>
  <div class="divider--half"></div>
  <div class="divider--half"></div>
  <div class="divider--half"></div>
  <div class="divider--half"></div>

> You may have noticed that in the image above the library also appear to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if you are running version 57 or later. The set up is very easy, you only need to open Indigo.Design and save it in your Sketch cloud. A detailed explanation of how to set Indigo.Design as cloud library can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the library successfully, it will show up in the **Symbols** section of the **Insert** menu in Sketch.

Reviewer: ```suggestion
You may have noticed that in the image above the library also appears to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if you are running version 57 or later. The set up is very easy, you only need to open the Indigo.Design Sketch file and save it in your Sketch cloud. A detailed explanation of how to set Indigo.Design as a cloud library can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the library successfully, it will show up in the **Symbols** section of the **Insert** menu in Sketch.
```

    This example is for an iPhone 11 in portrait orientation (414x896). If you don't see this preset artboard, choose Apple Devices from the dropdown at the top of the panel.

  <div class="divider--half"></div>

>   <img class="responsive-img" src="images/getting-started-artboard.png" srcset="images/getting-started-artboard@2x.png" />

Reviewer: This image is also huge, we need to downsize it to e.g. 70% so that it is more or less quite similar to the images that follow.

6.  **Add the inputs for user name and password.**

>     1.  Duplicate one of the inputs twice to have two more inputs.

Reviewer: ```suggestion
    1.  Duplicate one of the Name inputs twice to create two more inputs for Username and Password.
```
